client/components/sidebar.vue

```diff
@@ -43,6 +89,16 @@
 
   // @vue/component
   const component = {
+
+    filters: {
+      formatDate(value) {
```

Comment:
You have helper methods available here: https://github.com/peer/doc/blob/master/client/vue-extensions.js#L84

See how it is done here: https://github.com/peer/doc/blob/master/client/components/documents.vue#L20

(We could have the same, relative reactive time displayed, and if you hover you get a title with printed time.)